engine/schema/src/main/resources/META-INF/db/schema-41910to41920.sql

```diff
@@ -18,3 +18,6 @@
 --;
 -- Schema upgrade from 4.19.1.0 to 4.19.2.0
 --;
+
+-- Add last_id to the volumes table
+CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('volumes','last_id', 'bigint(20) unsigned DEFAULT NULL');
```

Comment:
```suggestion
CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.volumes', 'last_id', 'bigint(20) unsigned DEFAULT NULL');
```

I would specify the database as precaution